src/dataMigration.js

```diff
@@ -43,9 +44,12 @@ export const migrateDataToVersion = async (database, settings) => {
         compareVersions(fromVersion, migration.version) < 0 &&
         compareVersions(toVersion, migration.version) >= 0
       ) {
-        migration.migrate(database, settings);
+        migration.migrate(database, settings, fromVersion, toVersion);
       }
     }
+    database.write(() => {
+      createRecord(database, 'UpgradeMessage', fromVersion, toVersion);
```

Comment:
sorry Josh, the spec has changed.

can fromVersion and toVersion be integer..
have to export `versionToInteger` method from compareVersion.js

Reply:
i think it's also a good idea to send the text one (fromVersionText, toVersionText?)

Reply:
9f76aa7  then 119b91a